met-web/src/components/engagement/admin/create/authoring/AuthoringFeedback.tsx

```diff
@@ -0,0 +1,281 @@
+    };
+    const conditionalSelectStyles = {
+        width: '100%',
+        backgroundColor: colors.surface.white,
+        borderRadius: '8px',
+        boxShadow: '0 0 0 1px #7A7876 inset',
+        lineHeight: '1.4375em',
+        height: '48px',
+        marginTop: '8px',
+        padding: '0',
+    };
+    const toolbar = {
+        options: ['inline', 'list', 'link', 'blockType', 'history'],
+        inline: {
+            options: ['bold', 'italic', 'underline', 'superscript', 'subscript'],
+        },
+        blockType: { options: ['Normal', 'H2', 'H3', 'Blockquote'] },
+        list: { options: ['unordered', 'ordered'] },
+    };
+
+    const handleEditorChange = (newEditorState: EditorState) => {
+        const plainText = newEditorState.getCurrentContent().getPlainText();
+        setEditorState(newEditorState);
+        setBodyText(plainText);
+    };
+
+    const handleWidgetChange = (event: SelectChangeEvent<string>) => {
+        setCurrentWidget(event.target.value);
+    };
+
+    const handleRemoveWidget = () => {
+        if ('' === currentWidget) {
+            return;
+        } else {
+            setCurrentWidget('');
+        }
+    };
+
+    return (
+        <Grid container sx={{ maxWidth: '700px', mt: '1rem' }} direction="column">
+            <Grid item sx={{ mt: '1rem' }}>
+                <MetHeader3 style={metHeader3Styles}>Primary Content (Required)</MetHeader3>
+                <EyebrowText style={eyebrowTextStyles}>
+                    This section of content should provide a brief overview of what your engagement is about and what
+                    you would like your audience to do.
+                </EyebrowText>
+            </Grid>
+
+            <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.blue[10] }} item>
+                <label htmlFor="section_heading">
+                    <MetHeader3 style={metHeader3Styles}>
+                        Section Heading <span style={{ fontWeight: 'normal' }}>(Required)</span>
+                    </MetHeader3>
+                    <EyebrowText style={eyebrowTextStyles}>
+                        Your section heading should be descriptive, short and succinct.
+                    </EyebrowText>
+                    <TextField
+                        sx={{ backgroundColor: colors.surface.white }}
+                        value={sectionHeading}
+                        id="section_heading"
+                        counter
+                        maxLength={60}
+                        placeholder="Section heading message"
+                        onChange={(value) => {
+                            setSectionHeading(value);
+                        }}
+                    />
+                </label>
+            </Grid>
+
+            <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.gray[10] }} item>
+                <label htmlFor="body_copy">
+                    <MetHeader3 style={metHeader3Styles}>
```

Comment:
Even though this resembles a heading in the design, functionally it's more of a label. Same question as above, do screen readers get enough information about this field in order to fill it in?

Reply:
It would be read as a heading 3 at the moment, but that could definitely be changed if you think that's confusing to screen readers. Firefox doesn't have the best support for screen readers but I am testing the pages briefly with Pericles.

Reply:
I'd suggest using Mac OS' built-in Voiceover as it's one of the most used screen readers. It's really important that we test our work with a screen reader to ensure those users are having a good experience too

Reply:
Thanks, I tested with this too.